File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int n = s.length();
        int a = 0;
        int b=0;
        while(s[n-1] == ' '){
            n = n-1;
            s.resize(n);
        }
        for(int i=n-1; i>=0; i--){
            if(s[i] != ' '){
            a++;
            }

            else
            break;
        }
        return a;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int a=0;
        int n=s.length()-1;
        while(s.charAt(n)==' '){
            n--;
        }
        for(int i=n; i>=0; i--){
            if(s.charAt(i) != ' '){
            a++;
            }
            else
            break;
        }
        return a;
    }
}